Cargo.lock

```diff
@@ -908,6 +935,16 @@ name = "nodrop"
 version = "0.1.13"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
+[[package]]
+name = "nom"
+version = "5.0.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+dependencies = [
+ "lexical-core 0.4.3 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
would it be possible to disable `lexical` feature of `nom`? I think that should allow us to get rid of this dep

Reply:
I will check